types/cannon/index.d.ts

```diff
@@ -1027,6 +1062,7 @@ declare module CANNON {
 
     export interface ICollisionEvent extends IBodyEvent {
         contact: any;
+        target: any;
```

Comment:
ICollisionEvent and IBodyEvent aren't in the original JavaScript code since they are interaces.

but it comes from `dispatchEvent`
you can add an `addEventListener` to a body. 
when `dispatchEvent` is called, it sets `event.target`
https://github.com/schteppe/cannon.js/blob/569730f94a1d9da47967a24fad0323ef7d5b4119/src/utils/EventTarget.js#L92

If you console log a collision, you get a `CANNON.ICollisionEvent` which contains `body`, `contact`, `target` and `type`
![image](https://user-images.githubusercontent.com/6302260/127767600-934b49e4-cfd7-4dc0-a1f6-9a9fffee6730.png)
`target` is not in the current index.d.ts, 

But since you've pointed it out, I think it may be better as part of `IBodyEvent` 
 
```
    export interface IBodyEvent extends IEvent {

        body: Body;
        target: Body;
    }

    export interface ICollisionEvent extends IBodyEvent {
        contact: any;        
    }
```

Reply:
Hmm I see, thanks for the explanation :+1: